client/src/pages/private/ParticipantTable.js

```diff
@@ -500,7 +412,7 @@ export default () => {
       });
     };
     runAsync();
-  }, [pagination.currentPage, locationFilter, siteSelector, fsaFilter, lastNameFilter, emailFilter, order, tabValue]);
+  }, [pagination.currentPage, locationFilter, siteSelector, textState.emailFilter, textState.locationFilter, textState.lastNameFilter,  textState.fsaFilter, order, tabValue]);
```

Comment:
Is there any reason not to just pass in textState here instead of all the individual objects?

Reply:
This effect needs a ticket of it's own doesn't it 👀